scalding-hadoop-test/src/test/scala/com/twitter/scalding/reducer_estimation/ReducerEstimatorTest.scala

```diff
@@ -143,6 +180,23 @@ class ReducerEstimatorTest extends WordSpec with Matchers with HadoopSharedPlatf
 
     }
 
+    "run and catch memory source on right way" in {
```

Comment:
can you make this more descriptive? Maybe "Don't fail when a memory source is used" or "Ignore memory source in input size estimation" or something like that?